### ISSUE TITLE
week-00: gh-pages steps

### ISSUE THREAD

**Issue by hupili** [hupili/python-for-data-and-media-communication-gitbook#15]
@ChicoXYC here are two revisions.

* [x] https://github.com/hupili/python-for-data-and-media-communication-gitbook/blob/master/notes-week-00.md#step-4-commit--publish  . You showed a screenshot that is not part of the steps. The screenshot shows how to commit/ push to the current repo, i.e. `hupili/python-for-data-and-media-communication-gitbook`. Not the the repo you used to demo gh-pages. Please be consistent in the examples.
* [x] After step 4, please give a screenshot of the live website. It is also good to give the live URL. In this way, people know that it is working.

**Comment by hupili** [comment]
@ChicoXYC another thing to your demo page. Please make a more meaningful page. You don't have to use complex HTML/CSS or other templates. Having some real content is important. It can be a portfolio, piece of introduction, a small essay of learning Python, etc.

Say, just like what [JOUR2106](https://github.com/hkbu-jour2106/hkbu-jour2106.github.io) students can do after first lecture. Examples are like https://leungwasa.github.io/ and https://brittanyyy.github.io/ . I suggest to make a page that gives a basic description of COMM7780/JOUR7280 last term and links to your works, e.g. GitHub repos of homeworks/ projects, DNN blog post links, etc.

**Comment by ChicoXYC** [comment]
@hupili  update, change the wrong pic and build a mini cv in html. 80bb39f
